### ISSUE TITLE
tasks.js expects error.filename

### ISSUE THREAD

**Issue by startswithaj** [Semantic-Org/Semantic-UI#2971]
https://github.com/Semantic-Org/Semantic-UI/blob/master/tasks/config/tasks.js#L92

This expects that the error will have a filename property. Sometimes other errors can be thrown and caught by this. 

When it tries to access error.filename.match it throws.

Cannot call method 'match' of undefined suppressing the real error.

It should be 

if(error.filename && error.filename.match(/theme.less/)) {

**Comment by alxkolm** [comment]
Same problem.

**Comment by alxkolm** [comment]
> If you need to fix this force install gulp-autoprefixer to 2.3.1

How to do it?

**Comment by alxkolm** [comment]
Same problem.

Fast workaround:
```
rm -rf node_modules/gulp-autoprefixer
npm install gulp-autoprefixer@2.3.1
```

**Comment by jlukic** [comment]
When in doubt remove all the things. 

What version of autoprefixer is causing the above errors?

**Comment by startswithaj** [comment]
@jlukic gulp-autoprefixer 3.0.0 and 3.0.1 both have node 12 dependency. (running 3.0.1 on node 10 is what causes the problem). ~ This is speculation but If your not running npm version 2, I dont think npm version 1 inforces the node version dependency and allows 3.0.1 to install.

I just forced it to 2.3.1 which was the previous version my local npm was resolving prior to 3.0.1. 

Someone asked I think how I forced 2.3.1;  I just added gulp-autoprefixer 2.3.1 to my projects package.json which is a hackaround but because semantic will take any version its happy with this.

**Comment by jlukic** [comment]
I see. I'll try to check to see if there's anything I can do against issues when running autoprefixer 3.0.

**Comment by maxiaoxiong** [comment]
@alxkolm you are right!
thanks!

**Comment by TechnicalBro** [comment]
Just started using Semantic-UI in my pipeline today,and came across this issue. Installing gulp-autoprefixer@2.3.1 fixed it. Any chance of a global fix, or version update?

**Comment by r0bdiabl0** [comment]
Just hit with this bug myself, took a few days to find this issue but it fixed it. Please update, thanks!

**Comment by brandondrew** [comment]
Are we still required to use an ancient version of gulp-autoprefixer?  Is there any other fix for this?

**Comment by Dan1ve** [comment]
We run into this issue, too :-( I fix would be greatly appreciated, as a downgrade of autoprefixer to some ancient version is not an option for us

**Comment by CatalanCabbage** [comment]
This error still seems to be present, I encountered this in my GitLab-CI routine; any ideas on how to proceed?
I tried the workaround by @alxkolm, it didn't work.
For reference: submitted [new issue on Fomantic-UI](https://github.com/fomantic/Fomantic-UI/issues/1463) and a [question on SO](https://stackoverflow.com/questions/61780956/how-to-install-and-build-fomantic-ui-in-gitlab-ci)

**Comment by iamareebjamal** [comment]
Semantic UI is dead, please use fomantic UI

**Comment by CatalanCabbage** [comment]
Yup, I use Fomantic-UI; commented here just in case, since this bug has been carried over from Semantic to Fomantic.
Anyway, to anybody with this issue, for reference: a [PR has been submitted](https://github.com/fomantic/Fomantic-UI/pull/1469) with the fix, will be released in Fomantic 2.8.x :)
